MovieAppPetProject/MovieAppPetProject/Login/LoginViewController.swift

```diff
@@ -9,49 +9,35 @@ import UIKit
 
 class LoginViewController: UIViewController {
 
-    let username = "Admin"
-    let password = "DVTPassword"
+    // MARK: - Outlets
+    @IBOutlet private weak var usernameInput: UITextField!
+    @IBOutlet private weak var passwordInput: UITextField!
 
-    @IBOutlet weak var usernameInput: UITextField!
-    @IBOutlet weak var passwordInput: UITextField!
+    // MARK: - Variable
+    private let loginViewModel = LoginViewModel()
 
+    // MARK: - Functions
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        usernameInput.attributedPlaceholder = NSAttributedString(string: "Username", attributes: [NSAttributedString.Key.foregroundColor: UIColor.gray])
```

Comment:
do you have to use NSAttribute?
1)Can you not do this in storyboard?
2)If not, can you not just do 

usernameInput.text = "Test..."
usernameInput.textColor = UIColor.gray
and same for password

Reply:
I tried on storyboard and had a cll with Time we it isn't working in there. will refactor to your suggestion 💪

Reply:
there doesn't seem to be any methods for colour change on placeholder, and textColour changes only input text colour